.pylintrc

```diff
@@ -8,7 +8,6 @@ disable=
     logging-format-interpolation,
     logging-fstring-interpolation,
     no-member,
-    no-self-use,
```

Comment:
Maybe we should have put this in a separate branch, 'cause i have a feeling we have al 3 PRs making this change right now. :P